Plot/src/main/java/io/deephaven/plot/FigureWidget.java

```diff
@@ -19,10 +22,18 @@ public class FigureWidget extends FigureImpl implements LiveWidget, LiveWidgetVi
     private static final long serialVersionUID = 763409998768966385L;
     private String[] validGroups;
 
-    @SuppressWarnings("WeakerAccess") // this is used in the python integration
     public FigureWidget(final FigureImpl figure) {
         super(figure);
-        figure.getFigure().consolidatePartitionedTables();
+        getFigure().consolidatePartitionedTables();
```

Comment:
We said the management should move up to `FigureImpl`.

Reply:
..and have now changed our minds, going to leave it here, introduce a IsLivenessArtifact type, and have FigureWidget implement it.